app/views/orgs/lti_configurations/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<% organization = @organization || current_organization %>
```

Comment:
I think setting this variable within the view may be unnecessary, we could probably just use `current_organization` directly since it's guaranteed to be set by this point courtesy of the `before_action` on Orgs::Controller